docs/en/authorization_endpoint.rst

```diff
@@ -195,57 +172,35 @@ When the **scope** parameter is used, the following values are supported:
     - *email*;
     - *email_verified*.
```

Comment:
email_verified is only for cie

Reply:
https://github.com/italia/spid-cie-oidc-docs/pull/55/commits/0e038ebd85305c7f69305fc305dded8379de4022